rfcs/text/0002-rfc-environment.md

```diff
@@ -180,72 +225,11 @@ Stage 4: Document any new concerns and their resolution. The goal here is to eli
 Stage 4: Identify at least one real-world, production-ready implementation that uses these updated field definitions. An example of this might be a GA feature in an Elastic application in Kibana.
 -->
 
-* Elastic APM is already using `service.environment`
-
-```
-java -javaagent:/path/to/elastic-apm-agent-<version>.jar \
-     -Delastic.apm.service_name=my-application \
-     -Delastic.apm.environment=staging \
-     -Delastic.apm.server_urls=http://localhost:8200 \
-     -Delastic.apm.secret_token= \
-     -Delastic.apm.application_packages=org.example \
-     -jar my-application.jar
-```
-
-* Sample filebeat.yml using an `add_fields` processor.
-
-```yaml
-filebeat.inputs:
-  - type: log
-    enabled: true
-    paths:
-      - /var/log/my/file.log
-
-setup.template.settings:
-  index.number_of_shards: 0
-
-
-output.elasticsearch:
-  hosts: ["localhost:9200"]
-  protocol: "http"
-  username: "elastic"
-  password: "elastic"
-
-processors:
-  - add_host_metadata: ~
-  - add_cloud_metadata: ~
-  - add_docker_metadata: ~
-  - add_kubernetes_metadata: ~
-  - add_fields:
-      target: 'service'
-      fields:
-        environment: staging
-
-# Available log levels are: error, warning, info, debug
-logging.level: info
-
-```
 
 ## People
 
 
 * @cyrille-leclerc | author
 * ? | sponsor
-* @roncohen | subject matter expert
+* @carlos | subject matter expert
 * ? | grammar, spelling, prose
-
-
-## References
-
-<!-- Insert any links appropriate to this RFC in this section. -->
-
-### RFC Pull Requests
-
-<!-- An RFC should link to the PRs for each of it stage advancements. -->
-
-* Stage 0: https://github.com/elastic/ecs/pull/891
```

Comment:
Please add back that "References" section.

Note also that we'll want to add the link to this PR in the "RFC Pull Requests" section, below the "stage 0" entry: 

```
* Stage 1: https://github.com/elastic/ecs/pull/1070
```

Reply:
Problem solved.

Reply:
Can you add the link to the stage 1 PR as well? 🙏

Reply:
recursive link :-) I didn't think about it in this case but you are right :-)